python/lsst/pipe/analysis/coaddAnalysis.py

```diff
@@ -1362,6 +1374,39 @@ def matchCatalog(self, catalog, filterName, astrometryConfig):
         matches = matchNanojanskyToAB(matches)
         return joinMatches(matches, "ref_", "src_")
 
+    def plotRhoStatistics(self, catalog, plotInfoDict, zpLabel=None,
+                          forcedStr=None, postFix="", uberCalLabel=None):
+        """ Plot Rho Statistics with stars used for PSF modelling and
+        non-PSF stars.
```

Comment:
Please improve this doc string.

Reply:
It should have a yields rather than a returns.

Reply:
I don't quite understand the second comment on "yields rather than a returns". Also, the other functions do not have any doc string to go by. Can this all be taken care of in a dedicated ticket?